### ISSUE TITLE
Quicksilver Merchant Issue

### ISSUE THREAD

**Issue by Madbandi** [goatfungus/NMSSaveEditor#418]
Hi,

I have a problem about Quicksilver merchant. I want to buy an item but i can't. It says available. When I tried to buy, nothing happend. How can I solve this?
![Capture](https://user-images.githubusercontent.com/88367345/132257270-88e7f84f-5c17-4945-82c6-c4932af71abf.JPG)
![Capture2](https://user-images.githubusercontent.com/88367345/132257335-015d7032-18a3-4aa2-b7e5-8e0aa6bf8e29.JPG)

**Comment by wudilaojige** [comment]
got the same issue here,. Did u ever adjust the system time to accelerate some progress?

**Comment by Bertoo078** [comment]
Its mot because of the time acceleration i have it for a while and its because of the save editor ive learned the blueprints for it so u can place em unlimitly for free thats why i think we cant buy em cusz i can buy the newer ones i iddnt edit the blueprint in my save

**Comment by Bertoo078** [comment]
About that time issue ive fucked my settlement because of forwarding time and i cant fix it

**Comment by Madbandi** [comment]
> got the same issue here,. Did u ever adjust the system time to accelerate some progress?

No, i didn't :(

**Comment by wudilaojige** [comment]
> > got the same issue here,. Did u ever adjust the system time to accelerate some progress?
> 
> No, i didn't :(

most likely cuz we unlock all the blueprints, i might try delete the blueprints of sutff that i cant buy

**Comment by wudilaojige** [comment]
yep i deleted the void egg blueprints in editor then i can buy the egg now, but takes none quicksilver

**Comment by Madbandi** [comment]
> yep i deleted the void egg blueprints in editor then i can buy the egg now, but takes none quicksilver

Thank you, i will let you know when i try

**Comment by Mike-Blackwell** [comment]
> About that time issue ive fucked my settlement because of forwarding time and i cant fix it

More clues please what can't you fix/do?

**Comment by Mike-Blackwell** [comment]
I too had same issue before Frontiers came along, I had Expedition I & II awards that I legally received but when I went to the QS merchant I could see said rewards e.g. Stream Jetpack, Golden First Spawn Mask and fireworks but when I tried to select them they remained "Available". So based on this thread, I went back into NMS SaveEditor in the discoveries and removed said items which then allowed be to go back to the QS merchant and reclaim them and got the "previously awarded"  status, I then went back to NMS Save Editor and re-added the Discovery items, problem solved. I also found out the there are a few items without nomenclature e.g. streamline jetpack and golden first spawn Mask you will have to look at the game names to identify them.

**Comment by LiothGold** [comment]
> I too had same issue before Frontiers came along, I had Expedition I & II awards that I legally received but when I went to the QS merchant I could see said rewards e.g. Stream Jetpack, Golden First Spawn Mask and fireworks but when I tried to select them they remained "Available". So based on this thread, I went back into NMS SaveEditor in the discoveries and removed said items which then allowed me to go back to the QS merchant and reclaim them and got the "previously awarded" status, I then went back to NMS Save Editor and re-added the Discovery items, problem solved. I also found out the there are a few items without nomenclature e.g. streamline jetpack and golden first spawn Mask you will have to look at the game unique names to identify them.

...Maybe you could somehow do this to get the various Expedition rewards? I don't mean the banners and posters, but the ships and such.
